src/components/common/verify-code/verify-code-controller.ts

```diff
@@ -87,6 +90,7 @@ export function verifyCodePost(
           isConsentRequired: req.session.user.isConsentRequired,
           isLatestTermsAndConditionsAccepted:
             req.session.user.isLatestTermsAndConditionsAccepted,
+          isPasswordResetJourney: support2FABeforePasswordReset(),
```

Comment:
If it is the correct condition (it might well be), I think we should might want to call the parameter something different to `isPasswordResetJourney` (I know this is what is suggested in the ticket), since this to me implies something different to e.g. just `support2FABeforePasswordReset`

Reply:
Indeed it's name is a bit confusing, it was added as part of a previous user story as a feature switch for the 2fa before password reset, so I refrained from renaming it. But I agree it would be best to rename it.

Reply:
But I think the condition is correct, it's the naming that is confusing.

Reply:
Could we not rename `isPasswordResetJourney` though? That was introduced in this ticket, not a previous story, so it's up to us to name in here :-)

Reply:
Sorry, I remember it was added as part of a Gwyn's ticket for some reason, must've confused it with another variable. We can definitely rename it! :)